app/src/main/java/org/mozilla/tv/firefox/fxa/ADMService.kt

```diff
@@ -0,0 +1,12 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.fxa
+
+import mozilla.components.lib.push.amazon.AbstractAmazonPushService
+import com.amazon.device.messaging.ADMMessageReceiver
+
+class ADMService : AbstractAmazonPushService() {
```

Comment:
We need code to integrate the API key, right? File a follow-up.

Reply:
The api key is only needed for debug builds. Amazon provides an api key for the release builds. Where would be the best place to document for devs that you need to include the debug key? Is a comment on this class an appropriate place?

Reply:
Probably:
- Above the "SendTabFeature" declaration (i.e. where this key is required for the feature)
- In the readme
- Send an email to the team notifying them of this behavior